doc/dialects/exasol.md

```diff
@@ -129,6 +129,33 @@ USING SCHEMA_FOR_VS_SCRIPT.ADAPTER_SCRIPT_EXASOL WITH
 
 Note that you still need to provide a JDBC connection. This is used by the Virtual Schema adapter internally. It is not used for mass data transfer though. And that is where the performance gain comes from.
 
+## Transport Layer Security (TLS)
+
+With version 7.1 Exasol introduced TLS encryption on the database port. Other ports were TLS capable before that already.
+
+### Using TLS Connections
+
+To establish a TLS connection with an Exasol virtual schema, you must use Exasol Virtual Schema 5.0.3 or later and connect to an Exasol cluster running Exasol 7.1 or later. The reason why you need the Exasol Virtual Schema 5.0.3 or later is that with this version we built-in a JDBC driver that is TLS-capable.
```

Comment:
```suggestion
To establish a TLS connection with an Exasol Virtual Schema, you must use Exasol Virtual Schema 5.0.3 or later and connect to an Exasol cluster running Exasol 7.1 or later. The reason why you need the Exasol Virtual Schema 5.0.3 or later is that with this version we built-in a JDBC driver that is TLS-capable.
```

Reply:
> The reason why you need the Exasol Virtual Schema 5.0.3 or later is that with this version we built-in a JDBC driver that is TLS-capable.

not sure if that's relevant for users.

Reply:
As far as I can see from change log Exasol JDBC driver dependency was updated in version 5.0.4.